examples/sandbox_tmp/acados_ocp.json

```diff
@@ -1,6 +1,6 @@
 {
-    "acados_include_path": "/home/dangzilla/Documents/Programmation/acados/include",
-    "acados_lib_path": "/home/dangzilla/Documents/Programmation/acados/lib",
+    "acados_include_path": "/home/fb/devel/acados/include",
+    "acados_lib_path": "/home/fb/devel/acados/lib",
     "constraints": {
         "C": [],
         "C_e": [],
```

Comment:
Do not add anything from a sandbox please (if the folder is called sandbox and not sandbox_tmp, it will automatically be ignored)